app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,29 @@
+module ProjectsHelper
+  def project_teams(project)
+    Team.where.not(id: project.teams.select(:id)) 
+  end
+
+  def isTeamAvailable?(project)
+    Team.where.not(id: project.teams.select(:id)).to_a.empty?
+  end
+
+  def project_display_name(project)
+    display_name = nil
+    if project.project_category == 'TEAM'
+      display_name = 'TEAMS'
+    else
+      display_name = 'MEMBERS'
+    end
+    display_name
```

Comment:
Not needed.  Code returns value from line 13 and 15 even without this. We don't even need display_name. Refactor to ternary operator.